src/components/talks/talks.css

```diff
@@ -0,0 +1,39 @@
+.talks1 {
+	background-image: url("../../resources/talks/talkbg.png");
+	min-width: 100%;
+	min-height: 100%;
+	background-size: cover;
+}
+html,
```

Comment:
remove the styles from html and body, they're already defined in global stylesheets.